src/utils/coordinates/CoordinateSystem.class.js

```diff
@@ -22,13 +22,16 @@ export default class CoordinateSystem {
      *   re-projections.
      * @param {[Number, Number]} defaultCenter Coordinate to be used as the default center, will use
      *   the center of the bounds if none are given (and bounds are defined)
+     * @param {Boolean} isSwissProjection If this projection is made specifically to cover
+     *   Switzerland (if it uses our national LV95 or LV03 bounds, zoom level, etc...)
```

Comment:
I wounder if it would not make the code easier to understand if the `isSwissProjection` would be set in the constructor based on the `epsgNumber` when searching for it in the code it would be directly clear that it is only for lv03 and lv95 even though the name say it.

Reply:
See final comment about refactoring, this could be then part of `SwissCoordinateSystem` or `Lv95CoordinateSystem`